packages/contracts/src/errors.ts

```diff
@@ -45,6 +45,9 @@ export const Errors: ErrorsType = {
   "037": "NO_PROPOSAL",
   "038": "NO_OWNERSHIP_CHANGE",
   "039": "INITIATOR_MISMATCH",
+  "040": "INVALID_ROUTER_SIGNATURE",
```

Comment:
why not use `"022": "INVALID_SIGNATURE",`?

Reply:
agreed, we should either use invalid signature in all cases or remove 022 and replace it with invalid router sig / invalid user sig

obvi former is easier